packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -417,6 +448,22 @@ pool.addClient('app-client', {
 });
 ```
 
+All identity providers created in the CDK app are automatically registered into the corresponding user pool. All app
+clients created in the CDK have all of the identity providers enabled by default. The 'Cognito' identity provider,
+that allows users to register and sign in directly with the Cognito user pool, is also enabled by default. 
+Alternatively, the list of supported identity providers for a client can be explicitly specified -
+
+```ts
+const pool = new UserPool(this, 'Pool');
+pool.addClient('app-client', {
+  // ...
+  supportedIdentityProviders: [
+    UserPoolClientIdentityProvider.AMAZON,
```

Comment:
Holy shit those names :(

Reply:
Unfortunately namespacing. I don't want it conflicting with https://docs.aws.amazon.com/cognito/latest/developerguide/external-identity-providers.html